java/src/main/java/de/gdata/vaas/messages/VerdictRequestForUrl.java

```diff
@@ -26,22 +27,36 @@ public class VerdictRequestForUrl extends MessageType {
     @SerializedName("verdict_request_attributes")
     VerdictRequestAttributes verdictRequestAttributes;
     @Getter
-    @Setter
+    @SerializedName("use_shed")
+    transient boolean UseShed;
```

Comment:
transient?

Reply:
I would prefer not to call it `use_shed` but something like `use_hash_lookup` or something that does not leak internal component names and describes more what it does instead.